internal/services/recoveryservices/recovery_services_vault_resource.go

```diff
@@ -117,6 +117,12 @@ func resourceRecoveryServicesVault() *pluginsdk.Resource {
 				}, false),
 			},
 
+			"cross_region_restore": {
```

Comment:
could we make this consistent with other booleans in the provider?
```suggestion
			"cross_region_restore_enabled": {
```